Docker/ods-api-swaggerui/alpine/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN umask 0077 && \
     dos2unix /app/*.json && \
     dos2unix /app/*.sh && \
     rm -f /app/*.exe && \
-    apk del unzip dos2unix && \
+    apk del unzip dos2unix curl && \
```

Comment:
curl is only used with wget to fetch EdFi packages; once downloaded, the curl package is not needed. This also fixes one warning with the openssl version used by curl.